src/lib/device-types/sync/index.ts

```diff
@@ -0,0 +1,95 @@
+import * as _ from 'lodash';
```

Comment:
What's the point of making `sync/` a folder if it only has a single file?

Reply:
I plan on adding more (I have done so locally) as we switch the syncing to contracts, but I can also move it out of the folder for the time being if you prefer that

Reply:
Yes please, don't commit to future stuff. A rename from `device-types/sync.ts` to `device-type/sync/index.ts` is a trivial diff too.